content/blog/infrastructural.md

```diff
@@ -0,0 +1,78 @@
+
+Truly beautiful infrastructure undergirds the daily exchanges people actually care about—I don’t drink plumbing, I drink the water it carries and protects.
+
+Another way of thinking about “Infrastructure” (here defined loosely as “the *truly* essential structures that allow society at varying scales to operate”) is to think about it in terms of interpersonal support structure. 
+
+We deeply hope that Urbit can form a support structure for you and your community.
+
+These self-evident statements underlie the core problems we sought to address in designing a userspace for OS 1: How do we design an *actually* personal digital infrastructure? What does infrastructure look like when shared with immediate friends? With a cooperative organization? How can we impress upon Urbit's earliest adopters that once they've obtained an Urbit ID, their computer is at that moment *literally* entirely theirs?
+
+<br>
+
+<br>
+
+<img class="ba" src="https://media.urbit.org/site/posts/essays/infrastructural-2.jpg">
+
+## Expressing Infrastructure
+
+OS 1 was optimized from an interface design standpoint to ensure that a new inhabitant viscerally felt like there was a “space to inhabit”, or a “canvas to be painted”, or a set of simple and fundamental digital structures they could arrange to make their computer their own. 
+
+Counterintuitively, designing around this notion of a "rich blank slate” required OS 1 to be designed in a manner that *almost* results in it feeling incomplete, or on the verge of appearing deeply unremarkable. 
+
+A few key design decisions emphasized the above gestalt we sought to express:
+
+We decided very early on to revert the colorful presentation of Modulo to a literal blank slate—OS 1 is much like a sheet of graph paper now.
+All typography and typographic representation was severely flattened. This decision was one of our more drastic ones, and granted OS 1 a sense of being somewhat more "undirected" in the sense of its form. We hope that as our system becomes more robust, users can decide which elements of their systems require emphasizing, to feel most at home with their information.
+The majority of graphics used throughout the system were rendered into black and white line drawings. As architectural schematics and orthographic projections flatten their representation of reality so an architect can “see structure” easier, we made the decision to express a visual foundation that felt more like a diagram than an ergonomic user interface.
+A coupled notion of the above point is that we wanted to give our users the sense that modifying the underlying system was a simple task. Rather than present Urbit's first users with beautiful artistry and a sense that our infrastructure was to remain untouched and sacred, we decided to present a visual face to the project that felt nearly rudimentary, nothing outside the realm of editorial capability for the average person.
```

Comment:
These should probably be list items? I recall they were before.

Reply:
True — also realising the list of "beautiful infrastructure" points need to be bolded